ts/kpt-functions/src/types.ts

```diff
@@ -204,7 +202,8 @@ export class Configs {
    * @key key The key in the 'data' field in the ConfigMap object given as the functionConfig.
    */
   getFunctionConfigValue(key: string): string | undefined {
-    return this.getFunctionConfigMap().get(key);
+    const cm = this.getFunctionConfigMap();
+    return cm ? cm.get(key) : cm;
```

Comment:
cm ? cm.get(key) : undefined

it's more explicit

Reply:
Fix this in #176